ChangeLog

```diff
@@ -1,3 +1,19 @@
+Version 14.6.0
+--------------
+
+    * Fixed a file leak in please #1017
+    * Fixed a concurrency issue with uploading test results when using
+      --num_runs #1016
+    * Added metadata field to build rules that can be an arbitrary object.
+      This metadata can later be retrieved by a new BIF get_rule_metadata.
```

Comment:
nit: probably spell out "built-in function" in full rather than BIF (I'm not sure if readers will be familiar with that acronym)